tasks/managed/rh-sign-image/rh-sign-image.yaml

```diff
@@ -49,6 +49,10 @@ spec:
       type: string
       description: |
         The kubernetes secret to use to authenticate to Pyxis. It needs to contain two keys: key and cert
+    - name: batchLimit
+      type: string
+      description: The maximum param length of images to be processed in a batch
```

Comment:
the description should match with the one in the README.

Reply:
fixed